src/services/storage/handlers/offer.ts

```diff
@@ -72,6 +72,7 @@ const handlers: { [key: string]: Function } = {
     const flag = firstMsg.substring(2, 4)
 
     if (flag === '01') { // PeerId definition
+      const oldPeerId = offer.peerId
```

Comment:
I would rename to `previousPeerId` but its just a personal preference than "old" in this situation.

Reply:
Done